server/server.js

```diff
@@ -27,7 +27,8 @@ module.exports = async ({graphQLConfig, graphiqlConfig, postgresConfig, schemaLi
   app.use(expressPino)
 
   app.use('/graphql', bodyParser.json(), function (req, res, next) {
-    const graphql = graphqlExpress({schema, tracing})
+    const context = {request: req}
+    const graphql = graphqlExpress({schema, context, tracing})
```

Comment:
@aliok I feel like we will definitely need to refactor this eventually. Rebuilding the GraphQL Schema every time a request comes in will probably lead to really poor performance.

I'd like if we could open a ticket to resolve this issue separately

Reply:
@darahayes yeah, you're right. Go ahead and create the ticket :)